packages/ckeditor5-mention/tests/mentionui.js

```diff
@@ -131,6 +131,41 @@ describe( 'MentionUI', () => {
 		} );
 	} );
 
+	describe( 'contextual balloon in rtl', () => {
+		let getBalloonPanelPositionDataSpy;
+
+		beforeEach( async () => {
+			await createClassicTestEditor( { ...staticConfig } );
+
+			setData( model, '<paragraph>foo []</paragraph>' );
+			editor.locale.uiLanguageDirection = 'rtl';
+			getBalloonPanelPositionDataSpy = sinon.spy( mentionUI, '_getBalloonPanelPositionData' );
```

Comment:
Can we spy on `editor.plugins.get( 'ContextualBalloon' ).add()` instead? I'd rather avoid spying on private class members, they could change in an unpredictable way. Spying on the contextual balloon plugin provides a safe e2e test.